react/features/gene/GeneTable.js

```diff
@@ -42,7 +42,14 @@ export const getColumns = (gene, checked, handleCheck, handleCheckAll) => {
 	}
     ]
 
-    cols.push(...gene.columns.map(c => {
+    var gene_columns = gene.columns.reduce(function(result, col){
```

Comment:
This could be filter instead of reduce

Reply:
Thank you for your comments! Updated function accordingly.